### PR TITLE
Add Preset Save/Load UI

### LINKED ISSUE #10: Preset Save/Load UI
**Short-Term Goal, Feature**

We need some UI to save/load presets for easy use of plugin. 

------------------------------------------------

**Last Chat Log:**
[hooh] How should we approach to Saving/Loading Preset to remote location? I'm currently thinking about opening file explorer window to load or save presets.
What's your thoughts?

[ore] another thought on 1. perhaps even simpler is to provide a single line textbox for specifying a preset name next to a save button. then one can choose from a list of presets to load with a selector gui that behaves like the reflection probe selections. The height of the selectors would be bound by a vertical slider to prevent the window from growing too tall. how does that sound?

[hooh] Maybe like this? this also works.
image

[ore] What does save/load do compare with save defaults/load defaults? I assume this is without native file dialog, where do users type in name of presets?

From a screen estate perspective, I think we should have multiple columns as the preset usually has a short name and we should limit it to fit. Having one column would be a bit of waste of space. Have a look at Util.Selection() and where it’s being used at the moment. Just wrap a slider around it.

------------------------------------------------

Separated from #1, Added from #7

### PR DESCRIPTION
**Pull request by ore-**
fixes #10 
* this saves to /loads specific prefix and removed the default preset load. If we want a default preset to be loaded, we can add that as an option for the user in a subsequent change.

### CODE REVIEW

File: AI_Graphics/AI_Graphics/Preset.cs [hunk 1]
```diff
@@ -51,14 +51,11 @@ public struct Preset {
...
                     Load(bytes);
-                    Debug.Log(string.Format("Loaded preset file '{0}'", name + ".preset"));
                     return true;
                 } catch {
-                    Debug.Log(string.Format("Failed to load preset file '{0}'", name + ".preset"));
                     return false;
                 }
             } else {
-                Debug.Log(string.Format("Couldn't find preset file '{0}'", name + ".preset"));
                 return false;
```
Comment (hooh-hooah) [inline AI_Graphics/AI_Graphics/Preset.cs:59]: I think we should keep debug log to see what's going on in case when save/load is just not working

Comment (ore-) [reply AI_Graphics/AI_Graphics/Preset.cs:59]: unless you know we have existing load/save issues, I just tested it, the debug is spamming the log file on the success case. I tested a number of failure cases and think we have covered all the basis. at a minimum, we should remove the first debug log when loading is fine, ok with that?

Comment (hooh-hooah) [reply AI_Graphics/AI_Graphics/Preset.cs:59]: Yes seems fine

Comment (ore-) [reply AI_Graphics/AI_Graphics/Preset.cs:59]: let me put back the debug logs for failure cases.

**Pull request merged**